config/api.yml

```diff
@@ -461,6 +461,9 @@
         :identifier: miq_report_run
     :subcollections:
     - :results
+    :collection_actions:
+      :post:
+      - :name: import
```

Comment:
what's troubling me about the miq_report_export role identifier is that it doesn't require any extra authorization beyond `api_basic_authorize` - is that expected?